src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,8 +14,8 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
-          <h2 class="kuiLocalTab">
+        <header ng-if="!sectionName" id="tabHeader">
+          <h2>
```

Comment:
We should keep the class `kuiLocalTab` here. I think @timroes's comment was about removing the `id` and `tabindex` attributes. But I think we may actually want to revert this commit until @aphelionz has had a chance to explain how these attributes affect accessibility, since he's the one who established this pattern originally.